tests/run_test.sh

```diff
@@ -266,5 +266,30 @@ fi
 
 echo " $( date +%s )" >> ${LOG_DIR}/job_${JOB_NR}_timestamp.txt
 
+################################################################################
+# Remove RUN_DIRs if they are no longer needed by other tests
+################################################################################
+keep_run_dir=false
+if [[ ${TEST_NAME} == 'cpld_control_c96_p7' || ${TEST_NAME} == 'cpld_control_c192_p7' ]]; then
```

Comment:
Since those tests may be changed to a different name, we will have to change this script often. I'd suggest to use a more general way to decide which tests should be kept.

Reply:
I do not like all these special cases. Very difficult to maintain. run_test.sh should not have any code that depends on the test name. Variable $TEST_NAME is only used for log files. We should be able to rename all test to something else and everything should work.

Reply:
@junwang-noaa If the run directory from a test needs to be kept, can we just add a suffix to the test name? For example, cpld_control_c96_p7 becomes cpld_control_c96_p7.dep.

In run_test.sh, we only need the following:
if [[ ${TEST_NAME} == *.dep ]]; then
  keep_run_dir=true
fi

Reply:
@binli2337 I think this will add restriction on test name, and we don't know if there will be dependent test added later on. On the other hand, we already have the dependency information. The tests in the last column in rt.conf are the tests that other tests depends on and can only be deleted at the end of RT.

Reply:
@junwang-noaa The run_test.sh file is updated to use the information in the last column of rt.conf.